MANIFEST.in

```diff
@@ -4,11 +4,12 @@ include raiden_contracts/constants.py
 include raiden_contracts/data*/contracts.json
 include raiden_contracts/data*/deployment_kovan.json
 include raiden_contracts/data*/deployment_goerli.json
+include raiden_contracts/data*/deployment_goerli_unstable.json
 include raiden_contracts/data*/deployment_rinkeby.json
 include raiden_contracts/data*/deployment_ropsten.json
 include raiden_contracts/data*/deployment_mainnet.json
 include raiden_contracts/data*/deployment_services_kovan.json
-include raiden_contracts/data*/deployment_services_goerli.json
```

Comment:
This shoudn't be deleted, right?

Reply:
Definitively not. Fixed. Not my best day, apparently.

Reply:
Hehe, it's weekend soon!